arches_lingo/src/arches_lingo/components/scheme/editor/SchemeEditor.vue

```diff
@@ -17,37 +43,94 @@ const toggleSize = () => {
         emit("side");
     }
 };
+
+schemeComponents.map((x) => {
```

Comment:
I'm typically against single-letter declarations as it makes the code harder to read.